packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -565,29 +565,33 @@ describe('getInputProps', () => {
       });
     });
 
-    /* eslint-disable-next-line jest/no-done-callback */
-    test('lets user handle the errors', (done) => {
-      const getSources = jest.fn((..._args: any[]) => {
-        return [
-          createSource({
-            getItems() {
-              return new Promise<any>((resolve, reject) => {
-                reject(new Error('Fetch error'));
-              }).catch((err) => {
-                expect(err).toEqual(expect.any(Error));
-                done();
-                return [];
-              });
-            },
-          }),
-        ];
-      });
+    test('lets user handle the errors', async () => {
```

Comment:
I updated the test to make the linter pass (it didn't like the conditional `expect`).